docs/develop/node/running-a-node.md

```diff
@@ -185,6 +185,49 @@ Node is running!
 To check logs call: `nearup logs` or `nearup logs --follow`
 ```
 
+## Starting a node from backup
+Using data backups allows you to sync your node quickly by using public tar backup files. There are two types of backups for available for both `testnet` and `mainnet`:
+- regular 
+- archival 
+
+### Archive links
+
+`mainnet` 
+  - [regular](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar)
+  - [archival](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar)
+
+`testnet`
+  - [regular](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/rpc/data.tar)
+  - [archival](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar)
+
+Starting node using `neard` backup data 
+
+```bash
+./neard init --chain-id <chain-id> --download-genesis
+cd ~/.near/data
+wget <link-above>
+tar -xf data.tar
+rm data.tar
+./neard run
+```
+
+Starting node using `nearup` backup data:
+
+```bash
+nearup run <chain-id> && sleep 30 && nearup stop
```

Comment:
`sleep 30` won't be enough to download testnet genesis records, won't this break things?